src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -368,7 +371,8 @@ public int getTintColor(FluidState state, BlockAndTintGetter getter, BlockPos po
             .lightLevel(15)
             .density(3000)
             .viscosity(6000)
-            .temperature(1300)) {
+            .temperature(1300)
+            .addDripstoneDripping(0.05859375F, ParticleTypes.DRIPPING_DRIPSTONE_LAVA, Blocks.LAVA_CAULDRON, SoundEvents.POINTED_DRIPSTONE_DRIP_LAVA_INTO_CAULDRON)) {
```

Comment:
Magic numbers make me itch (part 2!). Based on [the Minecraft Wiki](https://minecraft.wiki/w/Pointed_Dripstone#Cauldrons), the percent chance for a cauldron being filled with lava by pointed dripstone stalactites is 15/256, which (according to my calculators) equals to this number exactly.

Reply:
I opted to use the LAVA_TRANSFER_PROBABILITY_PER_RANDOM_TICK in PointedDripstoneBlock for this instead so its less of a magic number. I'm currently patching it to be public, but if you would rather me use an AT for it, let me know.

Reply:
Same comment as above -- please use ATs instead of patches for modifying access.